src/pykeen/datasets/ogb.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+
+"""Load the OGB datasets.
+
+Run with python -m pykeen.datasets.ogb
+"""
+
+from typing import ClassVar, Optional
+
+import numpy as np
+
+from .base import LazyDataSet
+from ..triples import TriplesFactory
+
+__all__ = [
+    'OGBLoader',
+    'OGBBioKG',
+    'OGBWikiKG',
+]
+
+
+class OGBLoader(LazyDataSet):
+    """Load from the Open Graph Benchmark (OGB)."""
+
+    #: The name of the dataset to download
+    name: ClassVar[str]
+
+    def __init__(self, cache_root: Optional[str] = None, create_inverse_triples: bool = False):
+        self.cache_root = self._help_cache(cache_root)
+        self.create_inverse_triples = create_inverse_triples
+
+    def _load(self) -> None:
+        try:
+            from ogb.linkproppred import LinkPropPredDataset
+        except ImportError as e:
+            raise ModuleNotFoundError(
+                f'Need to `pip install ogb` to use pykeen.datasets.{self.__class__.__name__}.',
+            ) from e
+
+        self.dataset = LinkPropPredDataset(name=self.name, root=self.cache_root)
+        self.edge_split = self.dataset.get_edge_split()
+        self._training = self._make_tf(self.edge_split["train"])
+        self._testing = self._make_tf(
+            self.edge_split["test"],
+            entity_to_id=self._training.entity_to_id,
+            relation_to_id=self._training.relation_to_id,
+        )
+        self._validation = self._make_tf(
+            self.edge_split["valid"],
+            entity_to_id=self._training.entity_to_id,
+            relation_to_id=self._training.relation_to_id,
+        )
+
+    def _loaded_validation(self) -> bool:
+        return self._loaded
+
+    def _load_validation(self) -> None:
+        pass
+
+    def _make_tf(self, x, entity_to_id=None, relation_to_id=None):
+        triples = np.stack([x['head'], x['relation'], x['tail']], axis=1)
+
+        # FIXME these are already identifiers
```

Comment:
@mberr would be a good place to have a triples factory that could take pre-mapped triples

Reply:
cf. #57